source/plugins/go/src/network_flow_logs.go

```diff
@@ -0,0 +1,380 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"sync"
+	"time"
+	"strconv"
+	"encoding/json"
+
+	"github.com/fluent/fluent-bit-go/output"
+	"github.com/tinylib/msgp/msgp"
+	"github.com/google/uuid"
+)
+
+// Stream name for retina networkflow logs
+const RetinaNetworkFlowLogsStreamName = "RETINA_NETWORK_FLOW_LOGS"
+
+var (
+	// retina networkflow logs stream tag name
+	MdsdNetworkFlowLogsStreamTagName string
+	// flag to check whether the network flow logs are enabled
+	IsNetworkFlowLogsEnabled bool
+)
+
+var (
+	// networkflow telemetry
+    NetworkFlowTelemetryMutex = &sync.Mutex{}
+    NetworkFlowLogsMDSDClientCreateErrors float64
+    MdsdNetworkFlowClient net.Conn
+    NetworkFlowTagRefreshTracker time.Time
+)
+
+// NetworkFlowMsgPackEntry represents the object corresponding to a single messagepack event in the messagepack stream
```

Comment:
we should reuse the existing MsgPackEntry struct instead of introducing new one.

Reply:
existing mesgPack **Record** is map[string]string and it will not match the schema